developing/vcs/master-to-main.md

```diff
@@ -0,0 +1,125 @@
+# [Source Control](./README.md) / Master to Main branch rename
+
+These instructions will help moving repos from using the default branch name `master` and modify it to be
+`main`.
+
+<!-- toc -->
+
+* [Prereqs](#prereqs)
+* [Instructions to change default branch name](#instructions-to-change-default-branch-name)
+* [Future proofing](#future-proofing)
+* [Github](#github)
+* [CircleCI Changes](#circleci-changes)
+* [Terraform Registry](#terraform-registry)
+* [IDE Changes](#ide-changes)
+* [References](#references)
```

Comment:
Should we mention that danger `10.3.0` is necessary for `danger local` to work?

Reply:
Sure! Can you also send me the PR for that change?

Reply:
This is the [danger issue I created](https://github.com/danger/danger-js/issues/1057) which has a link to the PR that resolves it.